doc/modules/cassandra/pages/new/virtualtables.adoc

```diff
@@ -74,6 +74,8 @@ recent_hit_rate_per_second, recent_request_rate_per_second, request_count, and s
 
 |coordinator_write_latency |Records counts, keyspace_name, table_name, max, median, and per_second for coordinator writes.
 
+|cql_metrcs |Lists metrics for CQL prepared statements.
```

Comment:
I would make it consistent with the original metrics doc here - https://cassandra.apache.org/doc/latest/cassandra/operating/metrics.html
`Metrics specific to CQL prepared statement caching.`

Reply:
Also, there is a typo - should be `cql_metrics`

Reply:
Done